__tests__/app.test.js

```diff
@@ -69,3 +68,59 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("200 should add the comment inserted for the specified article_id", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "This is the new comment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment.body).toBe("This is the new comment");
+      });
+  });
+  test("400 should return an error if passed an empty body", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad Request");
+      });
+  });
+  test("404 should return an error if there is no article for this article_id", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "Hello, this is the new comment",
+    };
+    return request(app)
+      .post("/api/articles/100/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("There is no article");
+      });
+  });
+  test("404 should return an error if the specified user does not", () => {
+    const newComment = {
+      username: "Dennis-Tockan",
+      body: "Hello, this is the new comment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("Username not found");
+      });
+  });
+});
```

Comment:
Great tests Dennis, theres alot for this endpoint and you've only missed one. It's a normal 400 test that you've done before, if I make a post request to /api/articles/notanid/comments.